bravado/exception.py

```diff
@@ -2,7 +2,7 @@
 # -*- coding: utf-8 -*-
```

Comment:
Removed the line: `#!/usr/bin/env python`. 
Shouldn't we define the python encoding as `utf-8` instead of default ascii as the best practice?